modules/ob-release-notes-1-0.adoc

```diff
@@ -42,5 +42,6 @@ The `buildpacks` build strategy is currently in link:https://access.redhat.com/s
 [id="cli-new-features-1-0_{context}"]
 === CLI
 
+* With this release, you can use both `oc` and `shp` CLIs to work with {builds-shortname}.
```

Comment:
I am not sure if adding this line makes any change. I mean, we are releasing the CLI aka shp, which itself indicates that one can use `shp` or `oc` to manage the builds (Also "Work with Builds" section makes it evident). So, I think we can skip adding this line.
But, if we still feel that it's good to have this specified, feel free to proceed.

Reply:
removed this point, Thanks